rallyrolebot/cogs/rally_cog.py

```
@@ -32,3 +34,45 @@ async def cog_command_error(self, ctx, error):
     @commands.dm_only()
     async def set_rally_id(self, ctx, rally_id):
         data.add_discord_rally_mapping(ctx.author.id, rally_id)
+
+    @commands.command(name="price", help="Get the price data of a coin")
+    @validation.is_valid_coin()
+    async def price(self, ctx, coin_symbol):
+        def increase_decrease_gradient_index(percentage):
+            gradient = 5
+            if isinstance(percentage, float) and percentage != 0:
+                rank = math.floor(percentage / 20)
+                if(rank >= 5):
+                    rank = 5
+                elif(rank < -5):
+                    rank = -5
+                elif rank >= 0:
+                    rank += 1
+                gradient += rank
+            return gradient
+
+        price = coingecko_api.get_price_data(coin_symbol) or rally_api.get_price_data(coin_symbol)
+
+        percentage_24h = price["price_change_percentage_24h"]
+        percentage_30d = price["price_change_percentage_30d"]
+
+        if price is False:
+            await pretty_print(ctx, 
+                "There was an error while fetching the coin data", 
+                title="Error", 
+                color=ERROR_COLOR)
```

Should raise an error.
You can either create a custom error in [errors.py](https://github.com/CreatorCoinTools/RallyRoleBot/blob/master/rallyrolebot/errors.py) or use one from [discord](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#exceptions)